src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -137,6 +215,74 @@ class HeaderHelpMenuUI extends Component<Props, State> {
       </Fragment>
     ) : null;
 
+    let customContent;
+    if (helpExtension) {
+      const { appName, links, content } = helpExtension;
+
+      const getFeedbackText = () =>
+        i18n.translate('core.ui.chrome.headerGlobalNav.helpMenuGiveFeedbackOnApp', {
+          defaultMessage: 'Give feedback on {appName}',
+          values: { appName: helpExtension.appName },
+        });
+
+      const customLinks =
+        links &&
+        links.map((link, index) => {
+          const { linkType, title, labels = [], text, ...rest } = link;
+          switch (linkType) {
+            case 'documentation':
+              return this.createCustomLink(
+                index,
+                <FormattedMessage
+                  id="core.ui.chrome.headerGlobalNav.helpMenuDocumentation"
+                  defaultMessage="Documentation"
+                />,
+                index < links.length - 1,
+                {
+                  target: '_blank',
+                  rel: 'noopener',
+                  ...rest,
+                }
+              );
+            case 'github':
+              return this.createCustomLink(index, getFeedbackText(), index < links.length - 1, {
+                iconType: 'logoGithub',
+                href: this.createGithubUrl(labels, title),
+                target: '_blank',
+                rel: 'noopener',
+                ...rest,
+              });
+            case 'discuss':
+              return this.createCustomLink(index, getFeedbackText(), index < links.length - 1, {
+                iconType: 'editorComment',
+                target: '_blank',
+                rel: 'noopener',
+                ...rest,
+              });
+            case 'custom':
+              return this.createCustomLink(index, text, index < links.length - 1, { ...rest });
+            default:
+              break;
+          }
+        });
+
+      customContent = (
+        <>
+          <EuiTitle size="xxs">
+            <span>{appName}</span>
```

Comment:
Thoughts on changing this `<span>` to an `<h3>` and "Help" on line 314 (which I can't comment on) to an `h2`?
```suggestion
            <h3>{appName}</h3>
```

Reply:
Sure if you think those heading levels are appropriate even when the content of the page is unknown

Reply:
I think so because: 
- The `h1` should always be something about the page so we shouldn't be messing up the heading order
- I didn't want to skip any headings because, like you say, we don't know the rest of the page content
- These are custom, app specific help menus so I'm comfortable highlighting them to someone navigating by page headings
- Ultimately, these are actually pretty low risk/value either way because generally they won't even be rendered so it's mostly to let users quickly navigate this menu which isn't huge itself

So I think it's a nice little improvement without much risk but would also not fight too strongly for it if other people were opposed.

Reply:
Thank you for the explanation! I have updated them to use h2 and h3.